dv/uvm/data/sim.mk

```diff
@@ -33,7 +33,12 @@ gen_sv_flist: pre_compile ral
 
 compile: gen_sv_flist
 	@echo "[make]: compile"
+ifeq (${build_cmd},vlog)
+	cd ${sv_flist_gen_dir} && vlib work && ${build_cmd} ${build_opts}
```

Comment:
I'm happy either way, but I think a slightly cleaner way to do this would be something like:

``` make
compile-prep := true
ifeq ($(build_cmd},vlog)
  compile-prep := vlib work
endif

compile: gen_sv_flist
	@echo "[make]: compile"
	cd ${sv_flist_gen_dir} && $(compile-prep) && ${build_cmd} ${build_opts}
```
which avoids the weird indentation in the rule definition.